physionet-django/sso/templates/sso/login.html

```diff
@@ -21,8 +21,19 @@ <h6 class="card-subtitle mb-2 text-muted">Login through an external institute</h
           aria-disabled="true"
         >
           <i class="fa fa-university fa-lg mr-3"></i>
-          <span class="h6">{{ sso_login_button_text }}</span>
+          <span class="h6">login using you institution</span>
```

Comment:
Maybe we could use there a capital letter?

Reply:
It was a debug left over, changed it back.